OneMore/Commands/File/Markdown/MarkdownConverter.cs

```diff
@@ -141,8 +164,22 @@ public MarkdownConverter RewriteHeadings(IEnumerable<XElement> paragraphs)
 		}
 
 
+		/// <summary>
+		/// Applies standard OneNote styling to all recognizable headings in all Outlines
+		/// on the page
+		/// </summary>
+		public void RewriteTodo()
+		{
```

Comment:
Here also, should only affect Outlines that were affected, not all Outlines on page. User can apply custom style after if they want, otherwise abide by existing styles on page.